examples/vue/.browserslistrc

```diff
@@ -0,0 +1,2 @@
+> 1%
```

Comment:
Move example to another folder like already there are in master. I mean, we have in vanilla `examples/vanilla/js/validate-field`  or `examples/vanilla/ts/validate-field`